### PR TITLE
CORE-99 - Manage Waiting Lists

### PR DESCRIPTION
**Pull request by AxonC**

### CODE REVIEW

File: tests/Feature/Training/WaitingListFeatureTest.php [hunk 1]
```diff
@@ -74,29 +76,37 @@ public function testAStudentCanOnlyBeInAListOnce()
...
+        $this->waitingList->addToWaitingList($account, $this->privacc);
+        $this->waitingList->addToWaitingList($account2, $this->privacc);
+        $this->waitingList->addToWaitingList($account3, $this->privacc);
+
+        $this->actingAs($this->privacc)->post("nova-vendor/waiting-lists-manager/accounts/{$this->waitingList->id}/demote", [
             'account_id' => $account->id,
-            'position' => 1,
-        ])->assertRedirect(route('training.waitingList.show', $this->waitingList))
-            ->assertSessionHas('success', 'Waiting list positions changed.');
+        ])->assertSuccessful();
```
Comment (CalumTowers) [inline tests/Feature/Training/WaitingListFeatureTest.php]: Should we assert that an event was dispatched here?

Comment (AxonC) [reply tests/Feature/Training/WaitingListFeatureTest.php]: Is that should we in the sense that you don't think its required, or should we in that you think we haven't already?

Comment (CalumTowers) [reply tests/Feature/Training/WaitingListFeatureTest.php]: @AxonC We haven't already?

File: tests/Feature/Training/WaitingListFeatureTest.php [hunk 2]
```diff
@@ -74,29 +76,37 @@ public function testAStudentCanOnlyBeInAListOnce()
...
 
         Event::assertDispatched(AccountDemotedInWaitingList::class, function ($event) use ($account) {
```
Comment (AxonC) [inline tests/Feature/Training/WaitingListFeatureTest.php]: @CalumTowers *ahem*

**Pull request merged**